SeQuant/domain/eqs/single_ref_uccf12.h

```diff
@@ -84,6 +109,39 @@ class uccf12{
         }
       }
     }
+    else{return input;}
+  }
+  ExprPtr keep_up_to_2_body_terms(const ExprPtr& input) {
```

Comment:
@cmasteran1 use empty lines between functions just like you break paragraphs in text